src/wp-admin/setup-config.php

```diff
@@ -222,7 +222,7 @@ function setup_config_display_header( $body_classes = array() ) {
 		</tr>
 		<tr>
 			<th scope="row"><label for="pwd"><?php _e( 'Password' ); ?></label></th>
-			<td><input name="pwd" id="pwd" type="text" aria-describedby="pwd-desc" size="25" value="<?php echo htmlspecialchars( _x( 'password', 'example password' ), ENT_QUOTES ); ?>" autocomplete="off" /></td>
+			<td><input name="pwd" id="pwd" type="text" aria-describedby="pwd-desc" size="25" value="<?php echo htmlspecialchars( _x( 'password', 'example password' ), ENT_QUOTES ); ?>" autocomplete="new-password" /></td>
 			<td id="pwd-desc"><?php _e( 'Your database password.' ); ?></td>
```

Comment:
I'm not sure about this change. The database password is a current password, but not one that belongs to the individual filling out the form. 

It would be good to test this with some browsers and browser extensions such as LastPass and 1Password.

Reply:
I asked for some advice from a contact I have at 1Password to see if there's a practical way around this. I know you can have `autocomplete="section-some-section shipping|billing address"` so I am hoping something similar is available that may help here.

I'll let you know if I hear back anything of use.